docs/pages/installation/connect-requirements.md

```diff
@@ -0,0 +1,109 @@
+2) Once done, Cluster A connects to the API Server of cluster B  to configure the proper resources to establish a peering. This step requires also that Cluster B connects to the API server of cluster A to complete this phase.
+3) Once done, Cluster A connects to the Network Gateway of cluster B to establish a direct tunnel (i.e., VPN-like) for all the traffic between the two clusters.
+
+Although in some cases the step (3) may be completed successfully even if a single Network Gateway is publicly reachable (Liqo will try to establish the connection from A to B; if this fails, it forces the connection starting from B to A), we strongly suggest to publicly expose both Network Gateways in order to avoid problems in more advanced deployment options (e.g., a peering between A and B-C, followed by an automatic direct peering between B-C as a shortcut).
+
+
+### Examples of possible Liqo peering scenarios
+
+Liqo can be installed on different types of clusters, either on-premise or on public cloud providers.
+Therefore, a large number of peering scenarios are possible; among the most common options, we can cite:
+* public cloud (e.g., AKS, EKS, GKE) to public cloud
+* public cloud to on-premise
+* on-premise to on-premise
+* on-LAN to on-LAN (maily for testing purposes)
+
+In some cases, expecially on production setups, additional network limitations are present, such as firewalls that may impair network connectivity, which must be considered in order to enable Liqo peering.
+
+Finally, in some cases clusters may reside behind a NAT, which may also introduce additional limitations and it may require the configuration of some port-forwarding rules.
+
+#### Additional considerations
+
+The choice of the way you expose Liqo services to remote cluster may not be trivial in some cases.
+Here we list some additional notes you should consider in your choice:
+
+* **NodePort service**: although a NodePort service can be used to expose the Auth service and the Network gateway, often the IP addresses of the nodes are configured with private IP addresses, hence not being suitable for connections originated from the Internet. This happens rather often in production clusters, and on public clusters as well.
+* **Ingress controller**: in case the Auth service is exposed through an Ingress, you should remember that, by default, the Auth service uses the TLS protocol. Hence, either you configure your Ingress controller to connect to backend services with TLS as well, or you disable TLS on the Auth Service.
+* **API Server behind NAT**: in case the cluster is behind NAT, the IP address/port of the API server (i.e., `apiServer.address` in `values.yaml`) must be set as the _public_ <IP/port> tuple. This is not required for the Auth service, as the liqoctl command used to connect to the remote cluster 
```

Comment:
```suggestion
* **API Server behind NAT**: in case the cluster is behind NAT, the IP address/port of the API server (i.e., `apiServer.address` in `values.yaml`) must be set as the _public_ <IP/port> tuple.
```